lib/Invoice.php

```diff
@@ -55,14 +55,15 @@
  * @property int $starting_balance
  * @property string|null $statement_descriptor
  * @property string|null $status
- * @property mixed $status_transitions
+ * @property \Stripe\StripeObject $status_transitions
  * @property string|null $subscription
  * @property int $subscription_proration_date
  * @property int $subtotal
  * @property int|null $tax
- * @property mixed $threshold_reason
+ * @property float|null $tax_percent
```

Comment:
This is interesting

Reply:
yeah it's legacy/deprecated, but still supported for the foreseeable future